ts_scripts/backend_utils.py

```diff
@@ -6,21 +6,16 @@ def test_torchserve():
     # Lint Test
     print("## Started torchserve linting")
     ts_dir = os.path.join("ts", ".")
-    rc_file_path = os.path.join(".", "ts", "tests", "pylintrc")
-    py_lint_cmd = f"pylint -rn --rcfile={rc_file_path} {ts_dir}"
-    print(f"## In directory: {os.getcwd()} | Executing command: {py_lint_cmd}")
-    py_lint_exit_code = os.system(py_lint_cmd)
 
```

Comment:
Why lint is deleted?

Reply:
Currently, we're using a separate GitHub action for python linting https://github.com/pytorch/serve/blob/master/.github/workflows/lint.yml that relies on` pre-commit`. We already have some related PRs to disable pylint errors https://github.com/pytorch/serve/pull/1611 - if you'd rather I make a separate PR for this change lmk